front/src/components/molecules/TaxTable/TaxTable.tsx

```
@@ -16,23 +16,62 @@
+          {declarationResponse.products.map((detailedProduct) => {
+            console.log(
+              '🚀 ~ {declarationResponse.products.map ~ detailedProduct:',
+              detailedProduct,
+            );
+            return (
+              <TaxItem
+                detailedProduct={detailedProduct}
+                withCalculation={declarationResponse.canCalculateTaxes}
+                noDetails={noDetails}
+              />
+            );
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)